ecpy_hqc_legacy/tasks/tasks/instr/run_awg.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2016 by EcpyHqcLegacy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Tasks to set the parameters of arbitrary waveform generators.
+
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+
+from atom.api import (Unicode, Bool, set_default, Enum)
+
+from ecpy.tasks.api import (InstrumentTask, InterfaceableTaskMixin,
+                            validators)
+import logging
+
+
+class RunAWGTask(InstrumentTask):
+    """ Task to set AWG run mode
+
+    """
+    # switch to choose the AWG run mode: on or off
```

Comment:
For sphinx to properly pick up those description you need to use ```#:```. And a capital letter would be nice.

Reply:
ok